tools/openapi-schema/ansible-ai-connect-service.yaml

```diff
@@ -705,6 +705,18 @@ components:
         * `0` - ACCEPTED
         * `1` - REJECTED
         * `2` - IGNORED
+    Action693Enum:
```

Comment:
It's a shame we have to have a _system generated_ name for the enum.

Looks like we used to have `PlaybookGenerationActionActionEnum` (which was _slightly_ better).